ynr/apps/people/management/commands/people_clean_person_names.py

```diff
@@ -0,0 +1,35 @@
+from candidatebot.helpers import CandidateBot
+from django.core.management.base import BaseCommand
+from people.models import Person
+from popolo.models import OtherName
+
+
+class Command(BaseCommand):
+    help = "Replaces chars in person names that we don't want to be there."
+
+    def handle(self, *args, **options):
+        chars_to_replace = {"\xa0": " ", "\u200e": ""}
+
+        for char, replace in chars_to_replace.items():
+            qs = Person.objects.filter(name__contains=char).only("pk", "name")
+            for person in qs:
+                for other_name in person.other_names.all():
+                    other_name.name = other_name.name.replace(char, replace)
+                    other_name.save()
+                bot = CandidateBot(person_id=person.pk, update=True)
+                bot.edit_field("name", person.name.replace(char, replace))
+                bot.save(
+                    f"Replacing {repr(char)} with {repr(replace)} in person name"
+                )
+
+            qs = OtherName.objects.filter(name__contains=char)
+            for other_name in qs:
+                print(other_name)
+                existing_name = other_name.content_object.other_names.filter(
+                    name=other_name.name.replace(char, replace)
+                )
+                if existing_name:
+                    other_name.delete()
```

Comment:
We've got this chunk of code that I think is trying to de-dupe names, right? So if we had a person who has both "John Smith" and "John\xa0Smith" as alternate names, we will de-dupe them rather than save 2 identical alternate names. But should we be checking (both here and above) for the case where the `OtherName` is now a dupe of the primary name after fixing these issues?